src/presentation/test/integration/registration/general-partners.test.ts

```diff
@@ -39,4 +43,19 @@ describe("General Partners Page", () => {
     );
     expect(res.text).toContain(enTranslationText.generalPartnersPage.title);
   });
+
+  it("It should contain the proposed name - data from api", async () => {
```

Comment:
the "it" is already there

it("should contain the proposed name - data from api", async () => {

Reply:
removed the 'it'